scope/xgb.py

```diff
@@ -0,0 +1,206 @@
+import datetime
+import os
+import pathlib
+from .models import AbstractClassifier
+
+#!git clone https://github.com/DTrimarchi10/confusion_matrix.git
+import numpy as np
+from xgboost import XGBClassifier
+import xgboost as xgb
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import accuracy_score, confusion_matrix, matthews_corrcoef, mean_absolute_error
+from sklearn.metrics import f1_score, ConfusionMatrixDisplay
+#plot_confusion_matrix, is deprecated, so import ConfusionMatrixDisplay instead
+from sklearn.metrics import roc_curve, auc, precision_recall_curve, classification_report, average_precision_score
+import pandas as pd
+
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+import os
+import sys
+import copy
+import json
```

Comment:
Some of these imports aren't needed although they were included in the Jupyter notebook. We can clean them up as we continue to develop the code below.